packages/select/src/components/multi-select/multiSelect.tsx

```diff
@@ -352,6 +395,7 @@ export class MultiSelect<T> extends AbstractPureComponent<MultiSelectProps<T>, M
             // scroll active item into view after popover transition completes and all dimensions are stable.
             this.queryList.scrollActiveItemIntoView();
         }
+        this.input?.focus();
```

Comment:
can put this behind a conditional statement to only run if theres a child, but doesn't looks like this does anything to the current implementation

Reply:
if addressing above comment on `const input = (` this should be handled